components/friend.php

```diff
@@ -283,15 +289,18 @@ public function _list( $args, $assoc_args ) {
 	 * [--initiator=<user>]
 	 * : ID of the first user. Accepts either a user_login or a numeric ID.
 	 * ---
-	 * default: Random user.
+	 * default: If none is provided, a random user will be selected for each friendship.
```

Comment:
This `default` is interpreted as a literal string by wp-cli, which results in a "No user found..." error. Move the "If none is provided..." language to the short description and remove the `default`. (See `friend` as well.)